deploy/kubelet-config/01-kubelet-config.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: machineconfiguration.openshift.io/v1
 kind: KubeletConfig
 metadata:
-  name: system-reserved-worker
+  name: custom-kubelet
 spec:
-  autoSizingReserved: true
   machineConfigPoolSelector:
-    matchLabels:
-      pools.operator.machineconfiguration.openshift.io/worker: ""
+    matchExpressions:
+    - key: machineconfiguration.openshift.io/mco-built-in
```

Comment:
What is this label?

Reply:
Otherwise looks good.

Reply:
it's the common built in label for both the master and worker MCO groups: `oc get machineconfigpool --show-labels`

Reply:
This is safe for masters? I thought we were excluding masters?

Reply:
Yes, it is safe for masters. I confirmed that with the node team. We are not currently excluding masters for DPTP